trin-state/src/network.rs

```diff
@@ -64,41 +64,13 @@ impl StateNetwork {
                     debug!("Successfully bonded with {}", enr);
                     continue;
                 }
-                Err(OverlayRequestError::ChannelFailure(error)) => {
-                    debug!("Channel failure sending ping: {}", error);
+                // Tbh I'm a bit stumped on how to handle this area, so this is just a temporary
+                // solution to get the compiler passing. Will revisit if we decide to continue with
+                // anyhow
+                Err(msg) => {
```

Comment:
Yeah, I was stumped also here when trying prototyping it and that's why I assumed that getting rid of `OverlayRequestError` may not be so trivial.